frontend/src/pages/kakaoCallback/KakaoCallback.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../../common/components/AuthProvider/AuthProvider';
+import { postKakaoLogin } from './apis/postKakaoLogin';
+import { PAGE_URL } from '../../common/constants/url';
+import LoadingSpinner from '../../common/components/LoadingSpinner/LoadingSpinner';
+import { API_ENDPOINTS } from '../../common/constants/apiEndpoints';
```

Comment:
p3: 여기도 lint 적용되면 좋을 거 같아 ㅎㅎ API_ENDPOINTS는 여기서 쓰이지 않아서 이렇게 될 거 같아!

```suggestion
import { useEffect } from 'react';

import { useNavigate } from 'react-router-dom';

import { useAuth } from '../../common/components/AuthProvider/AuthProvider';
import LoadingSpinner from '../../common/components/LoadingSpinner/LoadingSpinner';
import { PAGE_URL } from '../../common/constants/url';

import { postKakaoLogin } from './apis/postKakaoLogin';
```